core/src/test/resources/sql/winagg.iq

```diff
@@ -839,5 +839,66 @@ EnumerableCalc(expr#0..3=[{inputs}], GENDER=[$t2], $1=[$t3])
 +--------+--------+
 (6 rows)
 
+!ok
+
+# Test excluding. Verified against Postgres
+!use scott
+
+select empno,
+  avg(comm) over (order by empno rows 2 preceding exclude current row) as e1,
+  avg(comm) over (order by empno rows 2 preceding exclude group) as e2,
+  avg(comm) over (order by empno rows 2 preceding exclude ties) as e3,
+  avg(comm) over (order by empno rows 2 preceding exclude no others) as e4
```

Comment:
Can you also add the same line but without "exclude no others" to verify they results in the same resultset? (here and in similar added tests)